ETWInsights/etw_reader.vcxproj

```diff
@@ -0,0 +1,214 @@
+      <InterfaceIdentifierFileName>%(Filename)_i.c</InterfaceIdentifierFileName>
+      <ProxyFileName>%(Filename)_p.c</ProxyFileName>
+    </Midl>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='RelWithDebInfo|Win32'">
+    <ClCompile>
+      <AdditionalIncludeDirectories>C:\src\UIforETW\ETWInsights\src;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AssemblerListingLocation>RelWithDebInfo/</AssemblerListingLocation>
+      <CompileAs>CompileAsCpp</CompileAs>
+      <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
+      <ExceptionHandling>Sync</ExceptionHandling>
+      <InlineFunctionExpansion>OnlyExplicitInline</InlineFunctionExpansion>
+      <Optimization>MaxSpeed</Optimization>
+      <PrecompiledHeader>NotUsing</PrecompiledHeader>
+      <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
+      <RuntimeTypeInfo>true</RuntimeTypeInfo>
+      <WarningLevel>Level4</WarningLevel>
+      <PreprocessorDefinitions>WIN32;_WINDOWS;_UNICODE;UNICODE;NDEBUG;CMAKE_INTDIR="RelWithDebInfo";%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <ObjectFileName>$(IntDir)</ObjectFileName>
+    </ClCompile>
+    <ResourceCompile>
+      <PreprocessorDefinitions>WIN32;_WINDOWS;_UNICODE;UNICODE;NDEBUG;CMAKE_INTDIR=\"RelWithDebInfo\";%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>C:\src\UIforETW\ETWInsights\src;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+    </ResourceCompile>
+    <Midl>
+      <AdditionalIncludeDirectories>C:\src\UIforETW\ETWInsights\src;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <OutputDirectory>$(ProjectDir)/$(IntDir)</OutputDirectory>
+      <HeaderFileName>%(Filename).h</HeaderFileName>
+      <TypeLibraryName>%(Filename).tlb</TypeLibraryName>
+      <InterfaceIdentifierFileName>%(Filename)_i.c</InterfaceIdentifierFileName>
+      <ProxyFileName>%(Filename)_p.c</ProxyFileName>
+    </Midl>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <ClInclude Include="C:\src\UIforETW\ETWInsights\src\etw_reader\etw_reader.h" />
+    <ClInclude Include="C:\src\UIforETW\ETWInsights\src\etw_reader\generate_history_from_trace.h" />
+    <ClInclude Include="C:\src\UIforETW\ETWInsights\src\etw_reader\stack.h" />
+    <ClInclude Include="C:\src\UIforETW\ETWInsights\src\etw_reader\system_history.h" />
+    <ClInclude Include="C:\src\UIforETW\ETWInsights\src\etw_reader\thread_history.h" />
+    <ClCompile Include="C:\src\UIforETW\ETWInsights\src\etw_reader\etw_reader.cc"  />
+    <ClCompile Include="C:\src\UIforETW\ETWInsights\src\etw_reader\generate_history_from_trace.cc"  />
+    <ClCompile Include="C:\src\UIforETW\ETWInsights\src\etw_reader\system_history.cc"  />
+    <ClCompile Include="C:\src\UIforETW\ETWInsights\src\etw_reader\thread_history.cc"  />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="C:/src/UIforETW/ETWInsights/base.vcxproj">

```

Comment:
Project references are also using absolute paths. I only discovered this and the other absolute paths because my UIforETW directory is in a different location.